rintoj/silomobile
# feat: add scan qrcode to login option

## Description

Adds the functionality to sign into the app by scanning a qr code.

### Change log

- Add scan to login modal 
- Add signInWithToken method to auth provider
- Fixed error handing when invalid token is passed

### How to test

- Tap on scan to login button on login screen
- Scan one of the qr codes below & verify the response

**Valid QR Code**
![auth](https://user-images.githubusercontent.com/6880914/157418826-f7165668-0cff-46ed-9d80-a12d6a422973.png)


**Invalid QR Code**
![auth_error](https://user-images.githubusercontent.com/6880914/157418820-b277853a-8b29-47f6-94bb-f9fb80503a49.png)
